<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDlg</class>
 <widget class="QDialog" name="DeleteDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete</string>
  </property>
  <widget class="QLabel" name="DeleteDlg_label_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제 ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeleteDlg_lineEdit_id">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>46</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteDlg_btn_Delete">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>44</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteDlg_btn_Close">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
